penguin/web/static/dashboard.html

```
@@ -193,16 +211,99 @@
+                    <input
+                        id="dashboard-auth-token"
+                        v-model="authToken"
+                        @keydown.enter.prevent="submitAuthToken()"
+                        type="password"
+                        autocomplete="off"
+                        spellcheck="false"
+                        placeholder="Paste local startup token"
+                        class="auth-input"
+                    />
+                    <button
+                        @click="submitAuthToken()"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)